step/src/test/java/com/google/sps/DatabaseHandlerTest.java

```diff
@@ -16,44 +16,108 @@
 
```

Comment:
All of your test cases are good and show that your methods are designed intuitively :)

Reply:
Thanks Adam!